packages/amplify_core/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.4.0
+
+### Features
+- feat(auth): TOTP MFA
+
+### Fixes
+- fix(analytics): AWSPinpointUserProfile Added null check for user attributes ([#3598](https://github.com/aws-amplify/amplify-flutter/pull/3598))
+- fix(auth): SDK exception mapping
```

Comment:
why do we have analytics here?

Reply:
Core+Flutter package aggregate all the changes

Reply:
if so there are missing some missing feat and fix

Reply:
I should've been more clear. Core captures all the Dart-only changes, Flutter captures all the Dart+Flutter changes.